multisigcore/providers/insight.py

```diff
@@ -23,11 +26,13 @@ def spendables_for_addresses(self, bitcoin_addresses):
             addresses = bitcoin_addresses[i:i+CHUNK_SIZE]
             url = "%s/api/addrs/%s/utxo" % (self.base_url, ",".join(addresses))
             r.extend(json.loads(urlopen(url).read().decode("utf8")))
-        spendables = []
+        spendables = {}
```

Comment:
I would like to stay compatible with pycoin for now, and this is an incompatible change.  The address can be calculated from the spendable's script.